osu.Game.Tests/Visual/Settings/TestSceneFileSelector.cs

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Allocation;
+using osu.Framework.Graphics;
+using osu.Game.Graphics.UserInterfaceV2;
+
+namespace osu.Game.Tests.Visual.Settings
+{
+    public class TestSceneFileSelector : OsuTestScene
+    {
+        [BackgroundDependencyLoader]
+        private void load()
+        {
+            Add(new FileSelector { RelativeSizeAxes = Axes.Both });
+        }
```

Comment:
Would be good to have coverage of both extensionless and extension-filtered views here.